gips/data/prism/prism.py

```diff
@@ -156,21 +156,10 @@ def __init__(self, filename):
         self.scale = scale
         self.version = version
         self.stability = stability
-        self.ver_stab = self._stab_score[self.stability] * .01 + int(self.version[1:])
+        self._version = self._stab_score[self.stability] * .01 + int(self.version[1:])
```

Comment:
Looks like `self.ver_stab` is used in `PrismAsset.query_provider`; I think it just needs to be changed to `self._version`.

Reply:
Good catch.

Reply:
resolved by 9e4230d